examples/neutron.rs

```diff
@@ -0,0 +1,187 @@
+use localic_utils::{
+    types::contract::{
+        AuctionStrategy, ChainHaltConfig, MinAmount, PairType, PriceFreshnessStrategy,
+    },
+    ConfigChainBuilder, TestContextBuilder,
+};
+use std::error::Error;
+
+/// Demonstrates using localic-utils for neutron.
+fn main() -> Result<(), Box<dyn Error>> {
+    env_logger::init();
+
+    let neutron = ConfigChainBuilder::default()
+        .with_admin_addr("neutron1kuf2kxwuv2p8k3gnpja7mzf05zvep0cyuy7mxg")
+        .with_chain_id("localneutron-1")
+        .with_denom("untrn")
+        .with_debugging(true)
+        .build()?;
+
+    // Create a testcontext
+    let mut ctx = TestContextBuilder::default()
+        .with_api_url("http://localhost:42069/")
+        .with_artifacts_dir("contracts")
+        .with_chain(neutron)
```

Comment:
if the `neutron` construction above would be a oneliner we could just put it here, something like:
```rust
...
.with_chain(ConfigChainBuilder::default_neutron().build())
...
```